crawl-ref/docs/develop/levels/guidelines.md

```diff
@@ -186,3 +187,10 @@ CHANCE directly in the vault. For ghost vault rooms for Vaults, instead set the
 Other tags we generally require for ghost vaults are `no_tele_into` and
 `no_trap_gen`. If you use `allow_dup` in your vault, also use
 `luniq_player_ghost` to avoid multiple vault placement on the same level.
+
+
+## Guidelines for no_tele_into
+
+The `no_tele_into` KPROP prevents teleports landing you on the tagged locations. Don't overuse this. Prevent egregiously unfair situations only.
+
+You don't need to tag the interior of all runed door / transporter vaults. For example, no ghost vaults use this KPROP.
```

Comment:
Let's first mention the places where it is required: areas you can't escape from, e.g. teleport closets or islands surrounded by impassable terrain. Ideally of course you should avoid creating such locations at all, but if you do create them they must have an escape hatch out (if that makes sense thematically) or a `no_tele_into`.

If you first explain what it's for, your followup of "but don't overuse it in cases xyz" will make more sense.